### ISSUE TITLE
hdf5 plugin - FieldTmp broken

### ISSUE THREAD

**Issue by PrometheusPi** [ComputationalRadiationPhysics/picongpu#1271]
The TmpField output seems to be broken in the current `dev`.
All value are zero.

This effects:
```
 - Density_{}
 - EnergyDensity_{}
 - ParticleCounter_{}
 - ParticleEnergy_{}
 - fields_ParticleLarmorEnergy_{}
```

I checked this with `h5py` amd `h5ls`. @psychocoderHPC seems to see the same problem in his current branch.

**Comment by psychocoderHPC** [comment]
I am on it I suspect #1243

**Comment by PrometheusPi** [comment]
@psychocoderHPC 
Might this be related to the changes in the event system in `refactore eventsystem` #1233 
where
```
__startAtomicTransaction( __getTransactionEvent( ) );
...
__setTransactionEvent( __endTransaction( ) ); 
```
is removed for `src/picongpu/include/fields/FieldTmp.tpp` and `src/picongpu/include/plugins/hdf5/restart/LoadSpecies.hpp`?

**Comment by PrometheusPi** [comment]
*Offline phone call with  @psychocoderHPC*: 
probably #1243 - #1233 should have no effect

**Comment by psychocoderHPC** [comment]
closed with #1272
